docs/source/release-process.rst

```diff
@@ -27,6 +27,14 @@ update the underline of the title to match the title's width.
 If you are renaming the release at this point, you'll need to git mv the file
 and also update releases.rst to reference the renamed file.
 
+To build and view the docs locally, run:
+
+.. code-block:: bash
+
+   $ cd docs
+   $ make html
+   $ open build/html/index.html
```

Comment:
open is osx specific, maybe suggest `${BROWSER} build/html/index.html` to open the index in the user's browser of choice